pkg/internal/ebpf/bhpack/hpack.go

```diff
@@ -0,0 +1,525 @@
+// Copyright 2014 The Go Authors. All rights reserved.
```

Comment:
Why copying the code instead of just doing:
```go
import "golang.org/x/net/http2/hpack"
```
?

Reply:
I had to vendor it because it was failing on unknown fields. We want to ignore the unknown so that we can continue parsing until we find `:method` and `:path`.